about.html

```diff
@@ -15,101 +15,100 @@
-						</svg>
-						<svg id="mobileClose" class="hidden" width="24" height="24" viewBox="0 0 24 24" fill="none"
-							xmlns="http://www.w3.org/2000/svg">
-							<g id="Group">
-								<g id="Group_2">
-									<path id="Vector"
-										d="M1.7416 23.9993C1.29575 23.9993 0.849903 23.8293 0.509942 23.4894C-0.169981 22.8094 -0.169981 21.7067 0.509942 21.026L21.0267 0.509942C21.7067 -0.169981 22.8094 -0.169981 23.4901 0.509942C24.17 1.18986 24.17 2.29265 23.4901 2.97327L2.97257 23.4894C2.63261 23.8293 2.18745 23.9993 1.7416 23.9993Z"
-										fill="white" />
-								</g>
-								<g id="Group_3">
-									<path id="Vector_2"
-										d="M22.2577 24C21.8118 24 21.366 23.83 21.026 23.4901L0.509942 2.97327C-0.169981 2.29334 -0.169981 1.19056 0.509942 0.509942C1.18986 -0.169981 2.29265 -0.169981 2.97327 0.509942L23.4901 21.0267C24.17 21.7067 24.17 22.8094 23.4901 23.4901C23.1494 23.83 22.7035 24 22.2577 24Z"
-										fill="white" />
-								</g>
-							</g>
-						</svg>
-					</button>
-
-					<div id="nav-links-container" class="hidden">
-						<ul id="nav-links">
-							<li>
-								<a href="/about.html"> About Us</a>
-							</li>
-							<li>
-								<a href="/impact.html"> Our Impact</a>
-							</li>
-							<li>
-								<a href="/values.html"> Our Values</a>
-							</li>
-							<li>
-								<a href="/events.html"> Events</a>
-							</li>
-							<li>
-								<a href="/team.html"> Our Team</a>
-							</li>
-							<li>
-								<a href="/contact.html"> Contact Us</a>
-							</li>
-						</ul>
-					</div>
-				</div>
-
-				<!-- hamburger menu -->
+		  <div class="dp-wrapper">
+			<!-- logo container -->
+			<div class="logo-container">
+			  <div class="img-container logo-icon">
+				<a href="/index.html">
+				  <picture>
+					<source media="(min-width:1180px)" srcset="/assets/icons/global/logo-icon-big.svg" />
+					<img src="/assets/icons/global/logo-icon.svg" alt="If These Lands Could Talk logo-icon"
+					  style="width: auto" />
+				  </picture>
+				</a>
+			  </div>
+			</div>
+			<!-- end logo -->
+			<!-- desktop menu -->
+			<div class="desktop-menu">
+			  <ul class="desktop-nav">
+				<li>
+				  <a href="/about.html"> About Us</a>
+				</li>
+				<li>
+				  <a href="/impact.html"> Our Impact</a>
+				</li>
+				<li>
+				  <a href="/values.html"> Our Values</a>
+				</li>
+				<li>
+				  <a href="/events.html"> Events</a>
+				</li>
+				<li>
+				  <a href="/team.html"> Our Team</a>
+				</li>
+			  </ul>
+			</div>
+			<a href="/contact.html" class="button2"> Contact Us</a>
+			<div id="menu-container">
+			  <button id="mobile-menu">
```

Comment:
This is still giving an error with the WAVE tool. I think we just need to add some descriptive text that is only visible to screen readers. Feel free to add an `.sr-only` class if we don't have one yet. 

<img width="1014" alt="Screen Shot 2024-01-30 at 8 07 38 AM" src="https://github.com/indigenousfriends/itlct-website-redesign/assets/704697/cc4479a6-aa17-46ee-a997-68dabba28153">

Reply:
resolved this issue